packages/base/src/constants.ts

```diff
@@ -40,6 +40,17 @@ export namespace CommandIDs {
   export const buffer = 'jupytergis:buffer';
   export const dissolve = 'jupytergis:dissolve';
 
+  // Layers only commands
+  export const newRasterLayer = 'jupytergis:newRasterLayer';
+  export const newVectorLayer = 'jupytergis:newVectorLayer';
+  export const newHillshadeLayer = 'jupytergis:newHillshadeLayer';
+  export const newImageLayer = 'jupytergis:newImageLayer';
+  export const newVideoLayer = 'jupytergis:newVideoLayer';
+  export const newShapefileLayer = 'jupytergis:newShapefileLayer';
+  export const newWebGlTileLayer = 'jupytergis:newWebGlTileLayer';
+  export const newHeatmapLayer = 'jupytergis:newHeatmapLayer';
+  export const newDrawVectorLayer = 'jupytergis:newDrawVectorLayer';
+
```

Comment:
Probably some rebase leftover

```suggestion
```

Reply:
Yes. I just realized before your comment that I had some rebase related issues.